security/AccessControl.rst

```diff
@@ -18,11 +18,22 @@ that a user or group may perform, whether to invoke a function or access
 data, are organized into a hierarchy, and privileges may be granted at
 any level.
 
+Users and Groups can be updated using the Users and Groups Kylo admin pages:
```

Comment:
This is actually an important detail that I was trying to figure out as part of helping somebody implement AD integration. If you're saying that the pages are disabled when you turn off auth kylo, then how are you supposed to create new groups and configure their access permissions? Unfortunately I'm not in a position to try out out myself before updating the docs. I will do my best to come back and update anything that turns out to be wrong when we have implemented it though.